src/components/NavBar/NavBarTools.tsx

```diff
@@ -1,16 +1,22 @@
-import { Link } from 'react-router-dom';
 import { tools } from '../../assets/pages';
+import NavBarToolItem from './NavBarToolItem';
 
-import './NavBar.css';
+interface ToolsProps {
+  platform: string;
+}
 
-const NavBarTools = () => (
+const NavBarTools = ({ platform }: ToolsProps) => (
   <div className="dropdown-menu">
     <div className="dropdown-menu-item">
-      {tools.map((tool) => (
-        <Link to={tool.link} key={tool.display}>
-          <span className="tool-item">{tool.display}</span>
-        </Link>
-      ))}
+      {tools
+        .filter((tool) => tool.tags.includes(platform))
```

Comment:
You're passing `platform` all the way from `main` to here and to other "child" components - it's time to introduce redux! 😄 




(But in the next PR though..... please)